README.md

```diff
@@ -237,3 +237,220 @@ Java 16 language specification: [see .pdf file](https://docs.oracle.com/javase/s
 
 ###[Ch. 18 - Type inference WIP](src/test/resources/test_ready/ch_18_type_inference)
 
+---
+
+## Examples of translation projections
+
+Bellow there are all designed mappings at the current moment.
+
+#### Identifiers
+Each identifier preserves name. There is no name mangling.
+
+### 15 Expressions
+
+---
+
+#### 15.8.1 Lexical Literals
+
+Now supported only integer, floating point and string literals:
+
+`1` -> 
+```
+[] > literal_1
+  prim__int.constructor_2 > @
+    prim__int.new
+    1
+```
+
+`1.0` ->
+```
+[] > literal_1
```

Comment:
Why is this one called `literal_1` and not `literal_1_0`? What if I have `1.23` literal? What about `1.56e-4`?

Reply:
This name is not connected with literal value. It's just unique identifier